app/bundles/CampaignBundle/Entity/LeadRepository.php

```diff
@@ -155,4 +155,38 @@ public function updateLead($fromLeadId, $toLeadId)
             $q->execute();
         }
     }
+
+    /**
+     * Check Lead in campaign.
+     *
+     * @param Lead  $lead
+     * @param array $options
+     *
+     * @return bool
+     */
+    public function checkLeadInCampaigns($lead, $options = [])
+    {
+        if (empty($options['campaigns'])) {
+            return false;
+        }
+        $q = $this->_em->getConnection()->createQueryBuilder();
+        $q->select('l.campaign_id')
+            ->from(MAUTIC_TABLE_PREFIX.'campaign_leads', 'l');
+        $q->where(
+                $q->expr()->andX(
+                    $q->expr()->eq('l.lead_id', ':leadId'),
+                    $q->expr()->in('l.campaign_id', $options['campaigns'])
+                )
+            );
```

Comment:
Move the 5 lines above 4 spaces to the left.

Use `$q->expr()->in('l.campaign_id', ':campaignIds')` and `$q->setParameter('campaignIds', $options['campaigns'], \Doctrine\DBAL\Connection::PARAM_INT_ARRAY)` to make this secure.

Reply:
Let's review now.